engines/bastion_katello/app/assets/javascripts/bastion_katello/products/details/repositories/new/new-repository.controller.js

```diff
@@ -96,7 +96,7 @@ angular.module('Bastion.repositories').controller('NewRepositoryController',
                 $scope.urlDescription = RepositoryTypesService.getAttribute($scope.repository, "url_description");
                 if ($scope.genericRemoteOptions && $scope.genericRemoteOptions !== []) {
                     $scope.genericRemoteOptions.forEach(function(option) {
-                        option.value = "";
+                        option.value = option.default || "";
```

Comment:
When `option.default` resolves to 0, JS considers it "falsey" and thus chooses empty string `""`.  In my testing, the Pulp remote gets sent empty string as the depth value for a new ostree repo with no depth explicitly set in the UI.